_posts/2020-09-08-communication-backend-comparison.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: Communication Backends, Raw performance benchmarking
+author: e_hoelzl
+published: true
+tags: [performance, results]
+excerpt_separator: <!--more-->
+---
+
+Distributed learning requires workers to collaborate by sharing learned information with their "colleagues". As MLBench
```

Comment:
by `swiftly` sharing. `With the accelerating growth of model sizes in modern deep learning, this aspect gains even more importance`.